dql/cli.py

```diff
@@ -759,11 +759,37 @@ def do_EOF(self):  # pylint: disable=C0103
         """Exit"""
         return self.onecmd("exit")
 
+    @repl_command
+    def do_clear(self):
+        """Clear the screen. Add a visual break (2 spaces and 1 horizontal rule)."""
+        return self._do_clear()
+
+    @repl_command
+    def do_cls(self):
+        """Clear the screen. Add a visual break (2 spaces and 1 horizontal rule)."""
+        return self._do_clear()
+
+    def _do_clear(self):
+        """
+        Clear the screen by scrolling window to place the input line at the top.
+        Does the following steps:
+        - Put 2 empty lines
+        - Draw a horizontal rule
+        - Clear the screen natively
+        - Remove "clear"/"cls" from history
+        """
+        print()
+        print()
+        console.rule()
+        os.system("cls" if os.name in ("nt", "dos") else "clear")
+        self.history_manager.remove_items(n=1)
+
     @repl_command
     def do_exit(self):
         """Exit"""
         self.running = False
         print()
+        self.history_manager.remove_items(n=1)  # remove "exit" from history
```

Comment:
This deletes an item even if I exit with CTRL+D. In the case where I didn't issue any commands, it throws an exception.

Reply:
Will fix.